edit-post/assets/stylesheets/main.scss

```diff
@@ -168,6 +168,32 @@ body.gutenberg-editor-page {
 			outline-offset: 0;
 		}
 	}
+
+	input[type=checkbox] {
+		border: 2px solid $dark-gray-300;
+		border-radius: $radius-round-rectangle / 2;
+		margin-right: 12px;
+		transition: none;
+
+		&:focus {
+			border-color: $dark-gray-300;
+			box-shadow: 0 0 0 1px $dark-gray-300;
+		}
+
+		&:checked:focus {
+			box-shadow: 0 0 0 2px $dark-gray-500;
+		}
+
+		&:checked {
+			background: theme( toggle );
+			border-color: theme( toggle );
+		}
+
+		&:before {
```

Comment:
Isn't this selector technically supposed to be `::before`? I know `:before` works but we should use `::before`: https://developer.mozilla.org/en-US/docs/Web/CSS/::before#Syntax

We should probs have a rule for it 🤷‍♂️

Reply:
It is, and I know, and I'm pretty sure we've discussed this. The reason I went with single colon (besides habits, but habits can change, you've seen this from me capitalizing and period ending my comments now), is that single colon is what we've used EVERYWHERE else in the code. If we add a linter rule and convert those all to double colons, that could probably be a good separate PR no?

Reply:
That's legit. Filed #8618.